local.env

```diff
@@ -0,0 +1,3 @@
+AIERA_SDK_API_URL=http://127.0.0.1:5000/graphql
```

Comment:
do you want to keep this?

Reply:
> do you want to keep this?

Yeah def

Reply:
didn't realize this was the local env file 👍